.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
+  "endOfLine": "lf",
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": false,
+  "singleQuote": true,
+  "overrides": [
+    {
+      "files": "*.scss",
+      "options": {
+        "tabWidth": 4
+      }
+    }
+  ]
+}
```

Comment:
Can we actually pull this whole config from our common eslint config at https://github.com/keep-network/eslint-config-keep ? https://github.com/keep-network/tbtc/blob/1149b2f0e1bf6d34c4f9897835699c406fdfe3fc/solidity/package.json#L54-L56 and https://github.com/keep-network/keep-ecdsa/blob/master/solidity/.eslintrc show how this looks in practice I believe.

Trying to keep our styles similar across the board. Tbh I want to make the eslintrc's strictly have `extends` lines, and I'm not sure we need to explicitly extend `prettier` as `eslint-config-keep` should already extend it.

Reply:
Oooh even better! Sorry I didn't know about this. I agree, the code should look as similar across the board as possible!